apollo-router/src/services/mod.rs

```diff
@@ -71,10 +71,8 @@ pub(crate) fn apollo_graph_reference() -> Option<String> {
 // set the supported `@defer` specification version to https://github.com/graphql/graphql-spec/pull/742/commits/01d7b98f04810c9a9db4c0e53d3c4d54dbf10b82
 pub(crate) const MULTIPART_DEFER_SPEC_PARAMETER: &str = "deferSpec";
 pub(crate) const MULTIPART_DEFER_SPEC_VALUE: &str = "20220824";
-pub(crate) const MULTIPART_DEFER_CONTENT_TYPE: &str =
-    "multipart/mixed;boundary=\"graphql\";deferSpec=20220824";
+pub(crate) const MULTIPART_DEFER_CONTENT_TYPE: &str = "multipart/mixed;deferSpec=20220824";
```

Comment:
that string is also used to generate the `Content-Type` header of the response to defer queries, so the boundary should be present. Maybe make separate constants, one for the media type in the `Accept` header and error logs, one for the `Content-Type`?
There's the same issue for subscriptions AFAIK